docs/keepassxc-browser-manual.md

```diff
@@ -0,0 +1,333 @@
+
+### 2. Configuration and settings
+
+KeePassXC-Browser settings can be accessed by clicking the KeePassXC-Browser icon and selecting *Settings* from the popup.
+
+#### 2.1. Settings: General
+
+On the *General* settings page key features, for example autocomplete, password generator and auto-fill can be enabled or disabled.
+The changes are saved immediately.
+
+<div style="text-align: center;">
+<img alt="KeePassXC-Browser General settings page" style="max-width: 100%;"
+src="{{ site.baseurl }}/images/browser-migration/manual_general_settings.png">
+</div>
+
+#### 2.2. Settings: Connected databases
+
+On the tab *Connected databases* KeePassXC-Browser shows you which databases are currently paired with KeePassXC.
+
+Removing a connected database from KeePassXC-Browser **does not** remove the key from KeePassXC, but the connection will be restricted
+and no credentials can be received. A new connection with key pairing is needed.
+
+<div style="text-align: center;">
+<img alt="KeePassXC-Browser Connected databases settings page" style="max-width: 100%;"
+src="{{ site.baseurl }}/images/browser-migration/manual_connected_databases_settings_page.png">
+</div>
+
+#### 2.3. Settings: Custom credential fields
+
+The credential fields previously saved, as described in section [1.4. Choose custom credential fields for a page](#14-choose-custom-credential-fields-for-a-page),
+are visible on this tab.
+
+<div style="text-align: center;">
+<img alt="KeePassXC-Browser Custom credential fields settings page" style="max-width: 100%;"
+src="{{ site.baseurl }}/images/browser-migration/manual_custom_credential_fields_settings_page.png">
+</div>
+
+#### 2.4. Settings: Site preferences
+
+TODO.
+
+<div style="text-align: center;">
+<img alt="KeePassXC-Browser configuring String Fields" style="max-width: 100%;"
+src="{{ site.baseurl }}/images/browser-migration/manual_site_preferences.png">
+</div>
+
+### 3. KeePassXC browser integration settings
+
+To access these settings, open KeePassXC's settings and select *Browser Integration* from the left panel.
+
+#### 3.1. General
+
+The following settings are available on this tab:
+
+##### Request to unlock the database if it is locked
+If KeePassXC-Browser is requesting a database unlock from the popup's *Reopen database* button, KeePassXC is brought to the front. After the database is unlocked, KeePassXC will
+be put to the background. This is enabled by default.
+
+##### Match URL scheme (e.g., https://...)
+Credentials are matched using the exact URL scheme. This is enabled by default.
+
+##### Return only best-matching credentials
+Only credentials with a full matching URL will be returned. This is disabled by default.
+
+##### Sort matching credentials by title / username
+Credentials seen in KeePassXC-Browser's autocomplete or popup login dialog are sorted by username as a default. When sorting by title is used, the credentials are first
```

Comment:
"When sorting by title is used," → "When sorting by title," or "When 'sort by title' is used" (where "sort by title" would be the name of the option in the display)